examples/chart/teleport-cluster/templates/deployment.yaml

```diff
@@ -73,6 +73,10 @@ spec:
       {{- end }}
 {{- if .Values.initContainers }}
       initContainers: {{- toYaml .Values.initContainers | nindent 6 }}
+  {{- if .Values.extraEnv }}
+        env:
+        {{- toYaml .Values.extraEnv | nindent 8 }}
+  {{- end }}
```

Comment:
Not sure what the use-case for this is, but if I'm reading this correctly, this will only set `env` for last init container from the initContainers list. Unless this is rendered for each initContainer (but I don't see a loop)?

Reply:
Yeah, you're right, good catch - this was functionality that I kinda half-implemented. I've removed it in 675e21b0d8a081d64af46fab2a22d4079e029417 - people can set `env` as part of their `initContainer` spec if they desire.